app/views/admin/rewards/_form_codes.html.erb

```diff
@@ -0,0 +1,22 @@
+    <div class="d-flex justify-content-center">
```

Comment:
- formatting of this file looks weird
- the name is misleading - it is not a form

Reply:
I changed the formatting and file name